destination/config/config.go

```diff
@@ -0,0 +1,60 @@
+// Copyright © 2022 Meroxa, Inc. & Yalantis
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:generate paramgen -output=paramgen.go Config
+
+package config
+
+import (
+	"strings"
+
+	"github.com/conduitio-labs/conduit-connector-redshift/common"
+)
+
+// Config is a destination configuration needed to connect to Redshift database.
+type Config struct {
+	common.Configuration
+}
+
+// Validate executes manual validations beyond what is defined in struct tags.
+func (c *Config) Validate() error {
```

Comment:
this `Validate` method can be moved to validate the common configs, and we can just call it here. Then, we can use the same method inside the `Validate` for the source config, instead of duplicating the whole method again

Reply:
Done.